### ISSUE TITLE
ELI5 What do these settings do?

### ISSUE THREAD

**Issue by Citrus333** [dylanraga/win11hdr-srgb-to-gamma2.2-icm#48]
Maybe I'm dumb, but these directions feel very unclear.

What settings to do I use for each ICC profile?  What are the differences in each profile?  If I download the 400/80 profile do I HAVE to set SDR brightness to 80 for the preset to look right?  Does the 200/30 preset look the same as the 400/80 preset just not as bright?  Do I use AutoHDR with there profiles?  Why does each profile have a nit value if the peak brightness of all the profiles is still 800?

**Comment by durrburger93** [comment]
I've been wondering the same thing for a long time and still have no idea, it is definitely not explained fully because it's a bit unintuitive.

**Comment by merc4derp** [comment]
I'll try and answer.

First off, about these numbers. 400/80 for example is the same thing expressed in nits and percentage. 80 percent brightness under windows should correspond to 400 nits of brightness, assuming a properly calibrated monitor.

These profiles are premade assuming a typical 1000nits peak brightness monitor that's been set up in windows to an sdr brightness of whatever number the profile says.

If you happen to have a 1000nits monitor and you're happy working with one of the 4 preset brightness values provided, sure you can use them.

In most cases however you'd be better off creating a custom profile with the provided tool instead.

Usage in autohdr depends on taste, as everything around autohdr naturally. Ideally use rtx hdr instead which doesn't need this.

As for why they have a nit value, that's the so called paper white nits corresponding to the sdr brightness as set in windows. How bright should sdr white be basically. Stuff like text, UI, etc. Typically you'd have paper white of 140-250 nits depending on environment and taste, but your peak hdr nits can shoot way beyond that when needed.


TLDR; If you got an HDR1000 monitor, set in windows the same sdr brightness and use the profile. Otherwise, make your own profile.
